tequilapi/http_api_server.go

```diff
@@ -23,33 +24,38 @@ type apiServer struct {
 	listener      net.Listener
 }
 
+// NewServer creates http api server for given addres port and http handler
 func NewServer(address string, port int, handler http.Handler) APIServer {
 	server := apiServer{
 		make(chan error, 1),
-		ApplyCors(handler),
+		DisableCaching(ApplyCors(handler)),
```

Comment:
It would be nicer to seen headers being applied sequentially, not wrapped like this. How this would look if we would need to add 4x different sets of headers?

Reply:
Nice idea. It could be improvement with composer which takes request, a list of decorators and applies them in given order like this:
Decorate(originRequest , ApplyCors, DisableCaching ... etc... )